src/cloud-api-adaptor/test/e2e/versions.yaml

```diff
@@ -0,0 +1,10 @@
+test_images:
+  nginx:
+    registry: "quay.io/sjenning/nginx"
+    tag: "1.15-alpine"
+  busybox:
+    registry: "quay.io/prometheus/busybox"
+    tag: "latest"
+  curl:
+    registry: "quay.io/curl/curl"
```

Comment:
You can use the following for nginx  - quay.io/confidential-containers/test/nginx:latest
I created the multi-arch manifest including amd64 and s390x. We can keep it updated as needed for tests.

For unprivileged nginx you can use - "quay.io/nginx/nginx-unprivileged:latest"

Reply:
Updated. Thanks, sjeninng's nginx is getting old now!